spec/lib/api/redfish-1.0/system-spec.js

```diff
@@ -771,6 +783,10 @@ describe('Redfish Systems Root', function () {
     });
 
     it('should return a valid system', function() {
+        redfish.getVendorNameById.resolves({
+            node: node,
+            vendor: 'notDellAndCisco'
```

Comment:
Suggest to change it to undefined (as the mechanism in https://github.com/RackHD/on-http/pull/742/files#diff-3ef377bee23e7df190a0c8a2a9be21f6R215), or "other" is better than "notDellAndCisco", in case other vendor will be added into this function, when "notDellAndCisco" could not cover all "other" cases.